pom.xml

```diff
@@ -16,10 +16,9 @@
         <cloverVersion>4.3.1</cloverVersion>
         <disableTestInjection>true</disableTestInjection>
         <hamcrest.version>2.2</hamcrest.version>
-        <java.level>8</java.level>
-        <jackson.version>2.13.4</jackson.version>
+        <jackson.version>2.14.2</jackson.version>
         <!-- When updating the Jenkins version, also update io.jenkins.tools.bom dependency management and the README -->
-        <jenkins.version>2.319.3</jenkins.version>
+        <jenkins.version>2.387.1</jenkins.version>
```

Comment:
It looks like Jenkins 2.375.4 also has the vuln fixes. In general when we bump our dependency I like to keep it lower where possible to make it maximally available to users, unless there's a compelling reason to go for 2.387? https://www.jenkins.io/changelog-stable/
Note that 2.375 looks to have been released in Nov of last year, and 381 in early march so there's not a _huge_ difference, so I'm not too fussed if this version has other advantages.

Reply:
I agree, lower is better, if they backported to  2.375.4 we should target that instead.

Reply:
I've changed the version to 2.375.4.